src/test/java/uk/gov/companieshouse/itemhandler/kafka/OrdersKafkaConsumerWrapper.java

```diff
@@ -77,9 +77,9 @@ public void orderProcessedException(final Exception x) throws Throwable {
      * emulates receiving of message from kafka topics
      * @param message
      */
-    @Around(value = "execution(* uk.gov.companieshouse.itemhandler.kafka.OrdersKafkaConsumer.*(..)) && args(message)")
+    @After(value = "execution(* uk.gov.companieshouse.itemhandler.kafka.OrdersKafkaConsumer.*(..)) && args(message)")
     public void aroundOrderProcessed(final Message message){
```

Comment:
If this is using `@After` instead of `@Around` then the name of the method should be afterOrderProcessed?

Reply:
Renamed